vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -37,6 +37,14 @@ class TextArea(VizroBaseModel):
     # See comment thread: https://github.com/mckinsey/vizro/pull/298#discussion_r1478137654
     _set_actions = _action_validator_factory("value")
 
+    @property
+    def _outputs(self) -> dict[str, str]:
+        return {"__default__": f"{self.id}.value"}
+
+    @property
+    def _inputs(self) -> dict[str, str]:
+        return {"__default__": f"{self.id}.value"}
```

Comment:
The current names feel a bit too vague to me so I would rename like this:

```suggestion
    @property
    def _action_outputs(self) -> dict[str, str]:
        return {"__default__": f"{self.id}.value"}

    @property
    def _action_inputs(self) -> dict[str, str]:
        return {"__default__": f"{self.id}.value"}
```